src/HelloTriangle.cpp

```diff
@@ -27,6 +27,9 @@
 
 #include <vulkan/vulkan.h> // also assume core+WSI commands are loaded
 static_assert( VK_HEADER_VERSION >= REQUIRED_HEADER_VERSION, "Update your SDK! This app is written against Vulkan header version " STRINGIZE(REQUIRED_HEADER_VERSION) "." );
+#if __APPLE__
+#include <vulkan/vulkan_beta.h>

```

Comment:
I've removed all the `#if __APPLE__` from the code. I've added `#include <vulkan/vulkan_beta.h>` in the code without a conditional as I'd like to avoid having to create a `#if VULKAN_PORTABILITY` in the code. Hopefully this is ok with you?

Reply:
It's fine for now. I guess it will need to be deleted\further updated when this functionality quits beta...

Although mildly annoying that all we need from the header is the macro string.